WMFComponents/Sources/WMFComponents/Components/Year in Review/WMFYearInReviewScrollView.swift

```diff
@@ -0,0 +1,81 @@
+import SwiftUI
+
+public struct WMFYearInReviewScrollView: View {
+
+    // MARK: - Properties
+    var primaryButtonAction: (() -> Void)?
+    var secondaryButtonAction: (() -> Void)?
+
+    @ObservedObject var appEnvironment = WMFAppEnvironment.current
+
+    @Environment(\.horizontalSizeClass) private var horizontalSizeClass
+
+    var sizeClassPadding: CGFloat {
+        horizontalSizeClass == .regular ? 64 : 32
+    }
+    
+    @ScaledMetric var scrollViewBottomInset = 125.0
+    
+    @State private var flashScrollIndicators: Bool = false
+    
+    var scrollViewContents: AnyView
+        var contents: AnyView?
+    
+    public init<ScrollViewContent: View>(
+        scrollViewContents: ScrollViewContent,
+        @ViewBuilder contents: () -> AnyView? = { nil }
+    ) {
+        self.scrollViewContents = AnyView(scrollViewContents)
+        self.contents = contents()
+    }
+
+    // MARK: - Lifecycle
+    
+    @available(iOS 17.0, *)
+    var flashingScrollView: some View {
+        ScrollView(showsIndicators: true) {
+            scrollViewContents
+                .padding(EdgeInsets(top: 0, leading: sizeClassPadding, bottom: scrollViewBottomInset, trailing: sizeClassPadding))
```

Comment:
@l-olson1214 I'm not sure why `WMFOnboardingView` used this large `scrollViewBottomInset`, but with your slides I think passing in `bottom: 0` here is working better. If you use the additional padding, your bottom toolbar thinks that some content is already displayed off screen, so a slight shadow appears at the bottom on some slides:

![screenshot](https://github.com/user-attachments/assets/2cbf9cfa-ba0c-4011-8b88-f6df6acf5b9a)

Reply:
Update: Actually I just noticed that you might need the larger content inset for the intro slide, which has buttons potentially covering up the scroll view content. So something to think about - if design review kicks this back due to the shadow, you may be able to fix it by making that `scrollViewBottomInset` property configurable from the outside and setting it to 0 for the slides with a bottom toolbar.